config/config.go

```diff
@@ -0,0 +1,101 @@
+// Copyright (C) 2024 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+// Package config provides functionality for processing Telemetry Agent configuration parameters.
+package config
+
+import (
+	"path/filepath"
+
+	"github.com/spf13/viper"
```

Comment:
Some time ago we choose  https://github.com/alecthomas/kong for CLI/Flags handling in Portal/PMM/Exporters. Is it possible to use it here as well?

Reply:
will check in next commits